--- atrace_35.rc	2025-04-17 13:30:45.462350300 +0000
+++ atrace_35_patched.rc	2025-04-17 13:31:51.663122000 +0000
@@ -539,7 +539,8 @@
     chmod 0440 /sys/kernel/tracing/hyp/events/hyp/host_mem_abort/id
 
 on property:persist.debug.atrace.boottrace=1
-    start boottrace
+    stop boottrace
+    setprop persist.debug.atrace.boottrace 0
 
 # Run atrace with the categories written in a file
 service boottrace /system/bin/atrace --async_start -f /data/misc/boottrace/categories
