--- atrace_31.rc	2025-03-15 13:51:21.380241100 +0000
+++ atrace_31_edited.rc	2025-03-15 13:52:19.036623400 +0000
@@ -339,7 +339,8 @@
     chmod 0666 /sys/kernel/tracing/instances/mm_events/per_cpu/cpu15/trace
 
 on property:persist.debug.atrace.boottrace=1
-    start boottrace
+    setprop persist.debug.atrace.boottrace 0
+    stop boottrace
 
 # Run atrace with the categories written in a file
 service boottrace /system/bin/atrace --async_start -f /data/misc/boottrace/categories
