--- atrace_32.rc	2025-04-17 13:55:41.895663500 +0000
+++ atrace_32_edited.rc	2025-04-17 13:56:19.146681400 +0000
@@ -389,7 +389,8 @@
     chmod 0666 /sys/kernel/tracing/instances/mm_events/per_cpu/cpu23/trace
 
 on property:persist.debug.atrace.boottrace=1
-    start boottrace
+    stop boottrace
+    setprop persist.debug.atrace.boottrace 0
 
 # Run atrace with the categories written in a file
 service boottrace /system/bin/atrace --async_start -f /data/misc/boottrace/categories
