--- bootchecker_34.rc	2025-04-17 14:05:14.957491600 +0000
+++ bootchecker_34_edited.rc	2025-04-17 14:05:38.808611600 +0000
@@ -1,13 +1,4 @@
 # bootchecker rc file, 
-on early-init
-    readahead /system/framework --fully
-    readahead /system/etc/sysconfig --fully
-    readahead /vendor/etc/sysconfig --fully
-    readahead /product/etc/sysconfig --fully
-    readahead /system/etc/permissions --fully
-    readahead /vendor/etc/permissions --fully
-    readahead /product/etc/permissions --fully
-
 service bootchecker /system/bin/bootchecker
     class late_start
     user system
