--- bootchecker_30.rc	2025-01-20 07:06:11.239526000 -0800
+++ bootchecker_30_edited.rc	2025-01-20 07:06:23.763279700 -0800
@@ -1,16 +1,5 @@
 # bootchecker rc file,
 
-on early-init
-    readahead /system/framework --fully
-	
-    readahead /system/etc/sysconfig --fully
-    readahead /vendor/etc/sysconfig --fully
-    readahead /product/etc/sysconfig --fully
-	
-    readahead /system/etc/permissions --fully
-    readahead /vendor/etc/permissions --fully
-    readahead /product/etc/permissions --fully
-
 service bootchecker /system/bin/bootchecker
     class late_start
     user system
