--- bootchecker_31.rc	2025-03-15 13:59:00.564849000 +0000
+++ bootchecker_31_edited.rc	2025-03-15 13:59:16.522500100 +0000
@@ -1,13 +1,4 @@
 # bootchecker rc file, 
-on early-init
-    readahead /system/framework --fully
-    readahead /system/etc/sysconfig --fully
-    readahead /vendor/etc/sysconfig --fully
-    readahead /product/etc/sysconfig --fully
-    readahead /system/etc/permissions --fully
-    readahead /vendor/etc/permissions --fully
-    readahead /product/etc/permissions --fully
-
 service bootchecker /system/bin/bootchecker
     class late_start
     user system
