--- bootchecker_33.rc	2025-04-17 14:02:21.935810000 +0000
+++ bootchecker_33_edited.rc	2025-04-17 14:02:55.375362900 +0000
@@ -1,13 +1,4 @@
 # bootchecker rc file, 
-on early-init
-    readahead /system/framework --fully
-    readahead /system/etc/sysconfig --fully
-    readahead /vendor/etc/sysconfig --fully
-    readahead /product/etc/sysconfig --fully
-    readahead /system/etc/permissions --fully
-    readahead /vendor/etc/permissions --fully
-    readahead /product/etc/permissions --fully
-
 service bootchecker /system/bin/bootchecker
     class late_start
     user system
