--- logd_29.rc	2025-03-15 14:28:17.196616600 +0000
+++ logd_29_edited.rc	2025-03-15 14:29:28.391546200 +0000
@@ -1,12 +1,13 @@
-# Debugging for SystemAnalysis
+# Horizon changes:
+# Disabled: Debugging for SystemAnalysis
 on early-init && property:ro.boot.debug_level=0x494d
-    setprop debug.enable true
+    setprop debug.enable false
 on early-init && property:ro.boot.debug_level=0x4948
-    setprop debug.enable true
+    setprop debug.enable false
 on early-init && property:ro.product_ship=false
-    setprop debug.enable true
+    setprop debug.enable false
 on early-init && property:ro.boot.force_upload=0x5
-    setprop debug.enable true
+    setprop debug.enable false
 
 service logd /system/bin/logd
     socket logd stream 0666 logd logd
@@ -40,5 +41,7 @@
     chown logd logd /dev/event-log-tags
     chmod 0644 /dev/event-log-tags
 
+# horizon changes:
 on property:sys.boot_completed=1
     start logd-auditctl
+    stop logd
