--- uncrypt.rc	2025-01-14 04:35:06.032374900 -0800
+++ uncrypt_edited.rc	2025-01-14 04:35:58.143929900 -0800
@@ -1,6 +1,4 @@
 on post-fs-data
-    # SAMSUNG
-    # create fota directory
     mkdir /data/fota 0770 system system
     mkdir /cache/fota 0770 system system
     chmod 0660 /cache/recovery/command
@@ -27,10 +25,4 @@
     class main
     socket uncrypt stream 600 system system
     disabled
-    oneshot
-
-service flash_recovery_sec /system/bin/install-recovery.sh
-    class main
-    user root
-    group root system cache
-    oneshot
+    oneshot
\ No newline at end of file
