--- wifiten.rc	2024-12-19 19:20:48.220572600 +0530
+++ wifiten_patched.rc	2024-12-19 19:22:35.060050500 +0530
@@ -1,4 +1,6 @@
-# Wi-Fi init file
+on boot
+    setprop wifi.interface wlan0
+
 on post-fs-data
     mkdir /data/vendor 0771 root system
     mkdir /data/vendor/log 0771 root system
@@ -7,37 +9,22 @@
     mkdir /data/vendor/wifi/wpa 0770 wifi wifi
     mkdir /data/vendor/wifi/wpa/sockets 0770 wifi wifi
     mkdir /data/vendor/log/wifi 0770 wifi system
-
-    # for WIFI TRP/TIS
-    chown wifi wifi /data/vendor/conn/.psm.info
     chmod 0640 /data/vendor/conn/.psm.info
-
-    # for WIFI Antenna Selection
-    chown wifi wifi /data/vendor/conn/.ant.info
     chmod 0640 /data/vendor/conn/.ant.info
-
-    # for WIFI Version
-    chown wifi wifi /data/vendor/conn/.wifiver.info
     chmod 0644 /data/vendor/conn/.wifiver.info
-
-    # for WIFI MEMDUMP
-    # The default value of below memdump file is set in SecProductFeature
-    #write /data/vendor/conn/.memdump.info 2
-    chown wifi wifi /data/vendor/conn/.memdump.info
     chmod 0640 /data/vendor/conn/.memdump.info
-
-    # for WIFI hotspot
-    chown wifi wifi /data/vendor/conn/.softap.info
     chmod 0644 /data/vendor/conn/.softap.info
-    chown system wifi /vendor/etc/wifi/indoorchannel.info
     chmod 0664 /vendor/etc/wifi/indoorchannel.info
-
-    # for WIFI MAC
-    chown wifi wifi /sys/wifi/mac_addr
     chmod 0640 /sys/wifi/mac_addr
+    chown wifi wifi /data/vendor/conn/.psm.info
+    chown wifi wifi /data/vendor/conn/.ant.info
+    chown wifi wifi /data/vendor/conn/.wifiver.info
+    chown wifi wifi /data/vendor/conn/.memdump.info
+    chown wifi wifi /data/vendor/conn/.softap.info
+    chown system wifi /vendor/etc/wifi/indoorchannel.info
+    chown wifi wifi /sys/wifi/mac_addr
 
-on boot
-    setprop wifi.interface wlan0
+# start of wifi daemons (except the testMode daemons)
 
 service wpa_supplicant /vendor/bin/hw/wpa_supplicant \
     -O/data/vendor/wifi/wpa/sockets -puse_p2p_group_interface=1 \
@@ -50,15 +37,6 @@
     disabled
     oneshot
 
-# start for WiFi MFG (TestMode)
-service mfgloader /system/vendor/bin/hw/mfgloader
-    class main
-    user wifi
-    group wifi inet net_raw net_admin
-    capabilities NET_ADMIN NET_RAW SYS_MODULE
-    disabled
-    oneshot
-
 service macloader /system/vendor/bin/hw/macloader
     class main
     user wifi
@@ -66,13 +44,4 @@
     capabilities NET_ADMIN NET_RAW SYS_MODULE
     oneshot
 
-service wlandutservice /system/bin/wlandutservice
-    class main
-    user system
-    group system wifi net_raw net_admin inet
-    capabilities NET_ADMIN NET_RAW
-    disabled
-    oneshot
-
-# end of WiFi MFG (TestMode)
-
+# end of wifi daemons (except the testMode daemons)
\ No newline at end of file
