--- dumpstate_30.rc	2025-01-20 07:09:07.847374600 -0800
+++ dumpstate_30_edited.rc	2025-01-20 07:10:59.033420400 -0800
@@ -4,18 +4,6 @@
     setprop init.svc.bugreportd stopped
     setprop init.svc.bugreportm stopped
 
-on property:init.svc.dumpstate=running
-    setprop dumpstate.is_running 1
-on property:init.svc.dumpstatez=running
-    setprop dumpstate.is_running 1
-on property:init.svc.bugreportd=running
-    setprop dumpstate.is_running 1
-on property:init.svc.bugreportm=running
-    setprop dumpstate.is_running 1
-on property:init.svc.dumpstate=stopped && property:init.svc.dumpstatez=stopped \
-    && property:init.svc.bugreportd=stopped && property:init.svc.bugreportm=stopped
-    setprop dumpstate.is_running 0
-
 on boot
     # Allow bugreports access to eMMC 5.0 stats
     chown root mount /sys/kernel/debug/mmc0/mmc0:0001/ext_csd
@@ -45,4 +33,21 @@
 service bugreportm /system/bin/dumpstate -m -d -z
     class main
     disabled
-    oneshot
\ No newline at end of file
+    oneshot
+
+# prevents dumpstate / log shits from starting, bugreports can't be taken cuz these are dead at that moment.
+on property:init.svc.dumpstate=running
+    setprop dumpstate.is_running 0
+    setprop init.svc.dumpstate stopped
+
+on property:init.svc.dumpstatez=running
+    setprop dumpstate.is_running 0
+    setprop init.svc.dumpstatez stopped
+
+on property:init.svc.bugreportd=running
+    setprop dumpstate.is_running 0
+    setprop init.svc.bugreportd stopped
+
+on property:init.svc.bugreportm=running
+    setprop dumpstate.is_running 0
+    setprop init.svc.bugreportm stopped
\ No newline at end of file
