--- retard.sh	2024-12-19 18:43:20.215664600 +0530
+++ retard_patched.sh	2024-12-19 19:14:28.480665800 +0530
@@ -1,9 +1,48 @@
 #!/system/bin/sh
 # Copyright (c) 2016, Samsung Electronics Co., Ltd.
 
-control=`getprop ro.vendor.use_data_netmgrd`
+is_integer() {
+    [[ "$1" =~ ^-?[0-9]+$ ]]
+}
 
-if [ "$control" = "true" ]; then
-      stop netmgrd
-      start netmgrd
-fi
+help_text() {
+    echo "###############################################"
+    echo -e " _   _            _                _   ___  __\n| | | | ___  _ __(_)_______  _ __ | | | \ \/ /\n| |_| |/ _ \|  __| |_  / _ \|  _ \| | | |\  /\n|  _  | (_) | |  | |/ / (_) | | | | |_| |/  \n|_| |_|\___/|_|  |_/___\___/|_| |_|\___//_/\_\ \n"
+    echo -e "###############################################\n"
+    echo " - Usage:"
+    echo "/system/etc/restart_radio.sh --stop-radio xx (duration in seconds)"
+    echo "/system/etc/restart_radio.sh --kill-radio-till-reboot"
+    echo "setprop ro.vendor.use_data_netmgrd true; /system/etc/restart_radio.sh"
+    echo -e "\nWhat this script does??" 
+    echo -e "    - This stops the netmgrd which is responsible for handling the radio process in the system.\n"
+    echo -e "What does \"--stop-radio\" do??"
+    echo -e "    - This prevents the netmgrd from starting for a given period of time.\n"
+    echo -e "What does \"--kill-radio-till-reboot\" do??"
+    echo -e "    - This stops the netmgrd from starting, but it doesn't mean that it won't start on boot.\n"
+}
+
+if [ "$(getprop ro.vendor.use_data_netmgrd)" == "true" ]; then
+    stop netmgrd
+    start netmgrd
+elif [ "$(echo "$1" | tr '[:upper:]' '[:lower:]')" == "--stop-radio" ]; then
+    if is_integer "$2"; then
+        stop netmgrd
+        sleep $2
+        start netmgrd
+    else 
+        help_text
+        return 1
+    fi
+elif [ "$(echo "$1" | tr '[:upper:]' '[:lower:]')" == "--kill-radio-till-reboot" ]; then
+    echo -e " - This shell will be doomed, please use another shell!"
+    while true; do
+        if [ -z "$(pidof netmgrd)" ]; then
+            sleep 10
+        elif is_integer $(pidof netmgrd); then
+            kill $(pidof netmgrd)
+        fi
+    done &
+    echo " - you may have seen an number above, please note it down and if you"
+    echo "   want to stop this script from killing the netmgrd, please enter the following in your console:"
+    echo "kill <the noted number here (without brackets)>"
+fi
\ No newline at end of file
